src/shared-components/alert/index.tsx

```diff
@@ -25,11 +25,24 @@ const alertIconMapping = {
   default: InfoIcon,
 };
 
-class Alert extends React.Component {
-  static Container = ({ children }) => (
-    <AlertsContainer>{children}</AlertsContainer>
-  );
+type AlertProps = {
+  avatarSrc?: string;
+  content: React.ReactNode;
+  ctaContent?: React.ReactNode;
+  duration?: string | number;
+  truncateText: boolean;
+  type: 'success' | 'error' | 'default';
+  onExit?: (rest: Omit<AlertProps, 'onExit'>) => void;
+};
+
+type AlertState = {
+  exiting: boolean;
+  exited: boolean;
+};
+
+type childrenProp = { children: React.ReactNode };
```

Comment:
Capitalize `childrenProp` to `ChildrenProp` both in definition and usage. (Alternatively, I would just inline the definition myself.)